api/response/schema.py

```diff
@@ -69,21 +94,93 @@ class Query(graphene.ObjectType):
     """
         Query to get the room response
     """
-    get_room_response = graphene.List(
-        Response,
+    get_room_response = graphene.Field(
+        PaginatedResponse,
         room_id=graphene.Int(),
+        page=graphene.Int(),
+        per_page=graphene.Int(),
         description="Returns a list of responses of a room. Accepts the arguments\
-            \n- room_id: Unique identifier of a room"
-    )
+            \n- room_id: Unique identifier of a room\
+            \n- page: Page number of responses\
+            \n- per_page: Number of room responses per page")
+
+    def map_room_responses(self, responses):
+        mapped_response = []
+        missing_resource = []
+        for response in responses:
+            response_id = response.id
+            suggestion = response.text_area
+            created_date = response.created_date
+            rating = response.rate
+            resolved = response.resolved
+            if len(response.missing_resources) > 0:
+                for resources in response.missing_resources:
```

Comment:
This should be resource and not resources

Reply:
Thanks. I'll fix that.